src/agent/src/uevent.rs

```diff
@@ -48,13 +74,23 @@ impl Uevent {
 
     // Check whether this is a block device hot-add event.
     fn is_block_add_event(&self) -> bool {
-        self.action == U_EVENT_ACTION_ADD
-            && self.subsystem == "block"
-            && self.devpath.starts_with(PCI_ROOT_BUS_PATH)
-            && self.devname != ""
+        #[cfg(target_arch = "aarch64")]
+        unsafe {
+            PCI_ROOT_BUS_PATH = create_pci_root_bus_path();
+        }
+        unsafe {
+            self.action == U_EVENT_ACTION_ADD
```

Comment:
Why unsafed this block?

Reply:
use of mutable static is unsafe and requires unsafe function or block